failing unit tests

### Luanti version

```true
5.15
```

### Operating system and version

Debian testing

### CPU model

_No response_

### GPU model

_No response_

### Active renderer

_No response_

### Summary

Building 5.15 on Debian, can you give me a hand with unit test failings?

```
======== Module TestActiveObject passed (0 failures / 1 tests) - 0ms
Catch test results: 
Randomness seeded to: 4120276995
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

 is a Catch2 v3.6.0 host application.
Run with -? for options

-------------------------------------------------------------------------------

test translations


  PO file parser

-------------------------------------------------------------------------------

./src/unittest/test_translations.cpp:84
...............................................................................


./src/unittest/test_translations.cpp:17: 

FAILED:


  REQUIRE( gamespec.isValid() )

with expansion:

  false

-------------------------------------------------------------------------------

test translations


  MO file parser

-------------------------------------------------------------------------------

./src/unittest/test_translations.cpp:102
...............................................................................


./src/unittest/test_translations.cpp:17: 

FAILED:


  REQUIRE( gamespec.isValid() )

with expansion:

  false

-------------------------------------------------------------------------------

x

-------------------------------------------------------------------------------

./src/unittest/test_irr_x_mesh_loader.cpp:21
...............................................................................


./src/unittest/test_irr_x_mesh_loader.cpp:26: 

SKIPPED:

-------------------------------------------------------------------------------

gltf

-------------------------------------------------------------------------------

./src/unittest/test_irr_gltf_mesh_loader.cpp:21
...............................................................................


./src/unittest/test_irr_gltf_mesh_loader.cpp:26: 

SKIPPED:


========



======================================================

=================
test cases:      9
 | 
     6 passed


 | 
1 failed


 | 
2 skipped

assertions: 155468
 | 
155466 passed


 | 
2 failed

++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
Unit Test Results: FAILED
    1 / 50 failed modules (1 / 320 failed individual tests).
    Testing took 5696ms total.
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
make[1]: *** [debian/rules:41: override_dh_auto_test] Error 1
make[1]: Leaving directory '/home/projects/debian/luanti'
make: *** [debian/rules:49: binary] Error 2
dpkg-buildpackage: error: debian/rules binary subprocess failed with exit status 2
```



### Steps to reproduce

```
git clone https://salsa.debian.org/dh/luanti/
cd luanti
dpkg-buildpackage -b -uc -us
```

--- a/src/unittest/test_translations.cpp
+++ b/src/unittest/test_translations.cpp
@@ -14,7 +14,8 @@
 static std::string read_translation_file(const std::string &filename)
 {
 		auto gamespec = findSubgame("devtest");
-		REQUIRE(gamespec.isValid());
+		if(!gamespec.isValid())
+			SKIP("devtest not found");
 		auto path = gamespec.gamemods_path + (DIR_DELIM "testtranslations" DIR_DELIM "test_locale" DIR_DELIM) + filename;
 		std::string content;
 		REQUIRE(fs::ReadFile(path, content));

